src/util/mongo.controller.js

```diff
@@ -73,11 +73,17 @@ const removeResource = async (id, resourceType) => {
   return collection.deleteOne({ id: id });
 };
 
+const findResourcesWithAggregation = async (query, resourceType) => {
```

Comment:
Can you add a JSDoc here explaining what doing the aggregate does for this query?

Reply:
Yep! Totally missed this when commenting.